3_Development/git.md

```diff
@@ -13,40 +13,68 @@ The commit message should look like this:
 > More detailed explanatory text. Wrap it to 72 characters.
 > The blank line separating the summary from the body is critical;
 > tools like GitHub can get confused if you omit it.
+
+
+In big repos, you can use a `[tag]` prefix in the summary, if you wish.
+
+
+The summary is all about "What does this commit do?" while the body is
+"Why does it do this?" and "Why was this approach chosen over others?"
+
+Further paragraphs come after blank lines.
+
+It is critical that you explain *why* you did the change. Don't state
+the obvious "File updated", "Text removed". Imagine you are reading
```

Comment:
Its's a bit ironic that your text says `Don't state the obvious "File updated"` but the title of the PR is `Update development/git section`, don't you think?

A better title would be `Expand on commit message/PR best practices`.

Reply:
Sorry for being such an *purist* so early in the morning btw :P

Reply:
I differentiate handbook (and other text repos like support) regarding commit messages. Why? because I already wrote all the things that I have changed and it feels duplicated content if I also write this in the commit message so I just add in the commit what section was updated.

That said I agree that we can and should try to continuously improve when it comes to writing good commit messages. I'll update the commit msg.